packages/ton-client/src/main.ts

```diff
@@ -1,11 +1,19 @@
 import './style.css'
```

Comment:
this file is becoming pretty messy, we really have to refactor it after the presentation

Reply:
I agree! Would like to do it in a pair programming session and learn something from you. Your code always looks so neat :)

Reply:
I agree! 
Would like to do it in a pair programming session and learn something from you. 
Your code always looks so neat :)

Reply:
thank you, yeah let's have a session after the presentation